src/Tizen.NUI/src/internal/Interop/Interop.Window.cs

```diff
@@ -311,6 +311,13 @@ internal static partial class Window
             [return: global::System.Runtime.InteropServices.MarshalAs(global::System.Runtime.InteropServices.UnmanagedType.U1)]
             public static extern bool IsWindowRotating(global::System.Runtime.InteropServices.HandleRef window);
 
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Window_SetPartialUpdateEnabled")]
+            public static extern bool SetPartialUpdateEnabled(global::System.Runtime.InteropServices.HandleRef window, bool enabled);
```

Comment:
I'm afraid it's a `void` type method in csharp-binder.

Reply:
Oh.. It was totally my mistake!! Thank you for your points. Fix done.